pooch/core.py

```diff
@@ -435,7 +442,7 @@ def get_url(self, fname):
         self._assert_file_in_registry(fname)
         return self.urls.get(fname, "".join([self.base_url, fname]))
 
-    def _check_download_hash(self, fname, downloaded):
+    def _check_download_hash(self, fname, downloaded, alg="sha256"):
```

Comment:
Is the default here needed since `hash_alg` should be set already if the code in `__init__` is correct?